framework/doc/content/source/vectorpostprocessors/VectorMemoryUsage.md

```diff
@@ -22,8 +22,10 @@ This VectorPostprocessor can be visualized using the
 [`VectorPostprocessorVisualizationAux`](/VectorPostprocessorVisualizationAux.md)
 AuxKernel.
 
+
```

Comment:
I don't think we need these extra new lines. You might as well just checkout the upstream revision of this

Reply:
ah my bad, was trying to fix something else at the same time but failed. I ll remove that